webapps/webapps.md

```diff
@@ -0,0 +1,40 @@
+# WebApps on IPFS
+
+This is a short guide on how to start making your own web applications on top of
+ipfs.  
+
+Ipfs provides an http api to interact with the daemon through, but by
+default it has a few security features enabled that make it difficult for people
+to use the api in their own apps. There are two different routes you could take
+to use the API, first, is to run your own webserver, and make http requests to
+localhost:5001. This is in my opinion the easier route for development. For this
+to work, you'll need to disable cross origin check by running the daemon with
+the environment variable `API_ORIGIN` set to `*`:
+
+```
+$ export API_ORIGIN="*"
+$ ipfs daemon
+```
+
+Now all of the requests you make to the api will work just fine.
```

Comment:
- `API_ORIGIN` is deprecated. use the config var (see `ipfs daemon`).
- we should not encourage `*`. it's a pretty big security hole. i can give you a single link that will delete all your pin set, gc, and then download a bunch of crap. So instead encourage to say: 

```
ipfs config --json API.HTTPHeaders.Access-Control-Allow-Origin \
  '["http://localhost:<port>", "http://127.0.0.1:<port>", "http://127.0.0.1:$desired-port"]'
```
`<port>` is for the local api port. and `$desired-port` is for user's port.